features/support/paths.rb

```diff
@@ -139,6 +139,9 @@ def path_to(page_name, *args) # rubocop:disable Metrics/AbcSize, Metrics/Cycloma
     when 'the provider dashboard'
       provider_admin_dashboard_path
 
+    when 'the new product page'
+      new_admin_service_path
+
```

Comment:
This path already exists: https://github.com/3scale/porta/blob/master/features/support/paths.rb#L250

Reply:
agreed